### ISSUE TITLE
dcos job --config-schema fails (binary DC/OS cli)

### ISSUE THREAD

**Issue by adragomir** [dcos/dcos-cli#839]
Please answer the following questions before submitting your issue. Thanks!

### What version of DC/OS + DC/OS CLI are you using (`dcos --version`)?

```
dcoscli.version=0.4.14
dcos.version=1.8.4
dcos.commit=e64024af95b62c632c90b9063ed06296fcf38ea5
dcos.bootstrap-id=5b4aa43610c57ee1d60b4aa0751a1fb75824c083
```

### What operating system and version are you using?

Mac OS X, 10.11.6

### What did you do?

Ran `dcos job --config-schema`

### What did you expect to see?

A JSON dump

### What did you see instead?

```
Traceback (most recent call last):
  File "cli/dcoscli/subcommand.py", line 99, in run_and_capture
  File "cli/dcoscli/job/main.py", line 28, in main
  File "cli/dcoscli/util.py", line 21, in wrapper
  File "cli/dcoscli/job/main.py", line 41, in _main
  File "dcos/cmds.py", line 43, in execute
  File "cli/dcoscli/job/main.py", line 149, in _job
  File "cli/dcoscli/job/main.py", line 645, in _cli_config_schema
  File "cli/env/lib/python3.4/site-packages/pkg_resources/__init__.py", line 1173, in resource_string
  File "cli/env/lib/python3.4/site-packages/pkg_resources/__init__.py", line 1605, in get_resource_string
  File "cli/env/lib/python3.4/site-packages/pkg_resources/__init__.py", line 1683, in _get
  File "/teamcity/buildAgent/work/a42c2156757d2b46/dcos-cli/cli/env/lib/python3.4/site-packages/PyInstaller/loader/pyimod03_importers.py", line 474, in get_data
FileNotFoundError: [Errno 2] No such file or directory: '/var/folders/mn/x3hkxh752nv7bfw13lwrwnz40000gr/T/_MEIK1bZho/dcoscli/data/config-schema/job.json'
```

**Comment by kensipe** [comment]
https://github.com/dcos/dcos-cli/pull/977

**Comment by kensipe** [comment]
This will be fixed on the next release of dcos-cli.  Thanks!!

**Comment by kensipe** [comment]
and it the output if you need it:

```dcos job --config-schema
{
  "$schema": "http://json-schema.org/schema#",
  "additionalProperties": false,
  "properties": {
    "service_name": {
      "description": "The name of the metronome cluster.",
      "title": "Name of the metronome cluster",
      "type": "string"
    },
    "url": {
      "description": "Api URL for talking to the Metronome scheduler.",
      "pattern": "^(?:(?:https?)://)(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\\.?)+(?:[a-zA-Z]{2,6}\\.?|[a-zA-Z0-9-]{2,}\\.?)?|\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(?::\\d+)?(?:/?|[/?]\\S+)$",
      "title": "Metronome service URL",
      "type": "string"
    }
  },
  "type": "object"
}```

**Comment by bamarni** [comment]
Thanks for reporting, this is fixed since some time now, #977 was a refactoring PR.
